cmd/bundle/generate/app.go

```diff
@@ -120,6 +124,16 @@ per target environment.`,
 		}
 
 		cmdio.LogString(ctx, "App configuration successfully saved to "+filename)
+
+		// If --bind flag is set, automatically bind the generated resource
+		bind, err := cmd.Flags().GetBool("bind")
+		if err != nil {
+			return err
+		}
+		if bind {
+			return deployment.BindResource(cmd, appKey, app.Name, true, false)
```

Comment:
This runs bundle initialization again, and potentially sets the same thing on the same context.

Risky?

Reply:
cc @denik

Reply:
I refactored it here https://github.com/databricks/cli/pull/3782/commits/0872fb0801c952dbedeb2bc7efde7c20848f4764